examples/container-service-extension/terraform.tfvars.example

```diff
@@ -0,0 +1,60 @@
+# Change configuration to your needs and rename to 'terraform.tfvars'
+
+# ------------------------------------------------
+# Provider config
+# ------------------------------------------------
+
+admin-user       = "******************************************"
+admin-password   = "******************************************"
+admin-org        = "System"
+vcd-url          = "https://myvcd.company.com/api"
+
+# ------------------------------------------------
+# CSE installation config
+# ------------------------------------------------
+
+# Org and VDC can be created or fetched. See 3.7-cse-install.tf for details
+org-name = "cse_org"
+vdc-name = "cse_vdc"
+
+storage-profile = "*" # Select a valid name for Storage Profile in System admin UI
+vdc-allocation = "AllocationVApp" # https://registry.terraform.io/providers/vmware/vcd/latest/docs/resources/org_vdc#allocation_model
+vdc-provider   = "nsxTPvdc1" # Select a valid name for Provider VCD in System admin UI
+vdc-netpool    = "NSX-T Overlay 1" # Select a valid name for Network Pool in System admin UI
+
+# Tier 0 configuration for creation
+tier0-manager        = "nsxManager1"
+tier0-router         = "VCD T0"
+tier0-gateway-ip     = "10.12.123.123"
+tier0-gateway-prefix = "19"
+tier0-gateway-ip-ranges = [
+  ["10.12.234.226", "10.12.234.228"],
+  ["10.12.234.230", "10.12.234.232"],
+  ["10.12.234.235", "10.12.234.237"]
+]
+
+# Edge gateway configuration for creation
+edge-gateway-ip     = "10.12.123.123" # Network subnet gateway ip, NOT primary IP
+edge-gateway-prefix = "19" # Prefix length
+edge-gateway-ip-ranges = [
+  ["10.12.234.226", "10.12.234.228"],
+  ["10.12.234.230", "10.12.234.232"],
+  ["10.12.234.235", "10.12.234.237"]
+]
+
+snat-external-ip       = "10.12.234.227" # Needs to be one IP from the available in the pool
+avi-virtual-service-ip = "192.168.8.88"
+
+# Routed network configuration for creation
+routed-gateway  = "192.168.7.1"
+routed-prefix   = "24"
+routed-ip-range = ["192.168.7.2", "192.168.7.100"]
+routed-dns      = ["8.8.8.8", "8.8.8.4"]
+
+# These elements will be fetched from VCD
+avi-controller-name  = "aviController1"
+avi-importable-cloud = "NSXT https://my-importable-cloud.company.com"
+
+# OVA name should be something like "ubuntu-2022-kube-v1.22.0+vmware.1-tkg.1-1234567891234567890.ova"
+tkgm-ova-name   = "ubuntu-2022-kube-v1.22.0+vmware.1-tkg.1-1234567891234567890.ova"
```

Comment:
is this name correct? as we testing with v1.21.2 version

Reply:
No, just a dummy example name. I was not sure whether to use a real OVA name as it maybe is something internal from Customer Connect.